runtime/permissions/lib.rs

```diff
@@ -1031,9 +1029,8 @@ impl NetDescriptor {
 
   pub fn parse_for_list(
     hostname: &str,
-    unstable_subdomain_wildcards: UnstableSubdomainWildcards,
   ) -> Result<Self, NetDescriptorParseError> {
-    Self::parse_inner(hostname, unstable_subdomain_wildcards)
+    Self::parse_inner(hostname, SubdomainWildcards::Enabled)
```

Comment:
Is this enum used anymore?

Reply:
Yes, wildcards should be disabled for parsing queried values.